app/templates/admin/edit/default.hbs

```diff
@@ -3,10 +3,10 @@
 
   <form>
     {{#each filteredColumns as |column|}}
-    <div class='model-column {{column}}'>
-        <label data-test={{concat 'form-label-' column}}>
-          {{column}}
-          {{admin-input record=this columnValue=column data-test=(concat 'edit-input-' column)}}
+    <div class='model-column {{column.key}}'>
```

Comment:
[nit][style] I know you didn't write this originally, but it looks like the indentation is off and needs to be indented one more level